### ISSUE TITLE
Setup instructions don't work on Ubuntu 20.04

### ISSUE THREAD

**Issue by cirego** [git-town/git-town#1345]
I was hoping to hack up a quick feature request for git-town, so I started following the [developer instructions](https://github.com/git-town/git-town/blob/6299fb922f68127fe44a4e7fd96b752bee4369ee/documentation/development/setup.md).

I ran into three issues. The first two I was able to resolve but the second one left me stuck as I haven't worked with Ruby in the past.

- `make: bundle: Command not found`: Fixed by running `sudo gem install bundler`
- `Could not find 'bundler' (1.17.2) required by your /home/chris/go/src/github.com/git-town/git-town/Gemfile.lock`. Apparently the version of Ruby shipping with 20.04 is ``2.1.4`. Fixed this by running `gem install bundler:1.17.2`.
- `Your Ruby version is 2.7.0, but your Gemfile specified 2.6.5`. Would 2.7.0 work or do I need to install 2.6.5?

I'm happy to open up a PR adding a note about installing `bundler:1.17.2`, but before I do, I thought I would ask if this is worth addressing?

**Comment by cirego** [comment]
Hi @kevgo! That's for the quick response! I'm now able to run the unittests, but I ran into a new issue:

```
[chris-work:~/go/src/github.com/git-town/git-town:(master[|])] make test
bundle exec cucumber_lint
/var/lib/gems/2.7.0/gems/cucumber_lint-0.1.2/lib/cucumber_lint/linter/feature_linter.rb:58: warning: Using the last argument as keyword parameters is deprecated; maybe ** should be added to the call
/var/lib/gems/2.7.0/gems/cucumber_lint-0.1.2/lib/cucumber_lint/linter/feature_empty_lines_linter.rb:5: warning: The called method `initialize' is defined here
/var/lib/gems/2.7.0/gems/cucumber_lint-0.1.2/lib/cucumber_lint/linter/feature_linter.rb:70: warning: Using the last argument as keyword parameters is deprecated; maybe ** should be added to the call
/var/lib/gems/2.7.0/gems/cucumber_lint-0.1.2/lib/cucumber_lint/linter/steps_linter.rb:5: warning: The called method `initialize' is defined here
/var/lib/gems/2.7.0/gems/cucumber_lint-0.1.2/lib/cucumber_lint/linter/feature_linter.rb:64: warning: Using the last argument as keyword parameters is deprecated; maybe ** should be added to the call
/var/lib/gems/2.7.0/gems/cucumber_lint-0.1.2/lib/cucumber_lint/linter/scenario_outline_linter.rb:7: warning: The called method `initialize' is defined here
/var/lib/gems/2.7.0/gems/cucumber_lint-0.1.2/lib/cucumber_lint/linter/feature_linter.rb:76: warning: Using the last argument as keyword parameters is deprecated; maybe ** should be added to the call
/var/lib/gems/2.7.0/gems/cucumber_lint-0.1.2/lib/cucumber_lint/linter/table_linter.rb:9: warning: The called method `initialize' is defined here
..............................................................................................................................................................................................................

206 files inspected (206 passed)
golangci-lint run --enable-all -D dupl -D lll -D gochecknoglobals -D gochecknoinits -D goconst -D wsl -D gomnd src/... test/...
tools/prettier/node_modules/.bin/prettier -l .
tools/text-runner/node_modules/.bin/text-run --offline
make: tools/text-runner/node_modules/.bin/text-run: Command not found
make: *** [Makefile:64: lint-md] Error 127
```

Not sure what the next step is here? I've done literally 0 work with node in the past.

**Comment by kevgo** [comment]
Sorry about that. I recently re-organized the developer tools and forgot to update the `make setup` script since I didn't need to run it. I just shipped #1359 to fix this. Once you `git sync`, and run `make setup` again, it should install all the Node dependencies. You need to have [Yarn](https://yarnpkg.com/getting-started/install) installed.

I have also removed the linter warnings in Ruby, so the tests should look presentable again.

**Comment by cirego** [comment]
Thanks for the quick reply! I was able to get past this step, but now I'm hitting an issue that's stumping me again.

When I run `make cuke`, this is what I get:

```
go install -ldflags "-X github.com/git-town/git-town/src/cmd.version=v0.0.0-test -X github.com/git-town/git-town/src/cmd.buildDate=today"
godog --concurrency=56 --format=progress features/git-town features/git-town-alias features/git-town-append features/git-town-config features/git-town-hack features/git-town-install-fish-autocompletion features/git-town-kill features/git-town-main_branch features/git-town-new-branch-push-flag
failed to compile testmain package: exit status 2 - output: /tmp/go-build586423073/b001/_testmain.go:4:2: can't find import: "github.com/git-town/git-town/vendor/github.com/cucumber/godog"

make: *** [Makefile:18: cuke-go] Error 1
```

To confirm, my checkout of `git-town` is inside of my `GOPATH`:

```
[chris-work:~/go/src/github.com/git-town/git-town:(master[|])] go env GOPATH
/home/chris/go
[chris-work:~/go/src/github.com/git-town/git-town:(master[|])] pwd
/home/chris/go/src/github.com/git-town/git-town
```

Am I missing something simple?

**Comment by kevgo** [comment]
Wow, you got a beefy machine, 56 cores?? Apologies for this issue. A few days ago I switched over to Go modules and forgot to update the setup instructions. Please move the repository outside the GOPATH. This will make the tests run. At least, now the developer instructions are up to date!

**Comment by kevgo** [comment]
Shipped https://github.com/git-town/git-town/pull/1371 that documents Go modules.

**Comment by cirego** [comment]
Perfect! Thanks for updating the instructions. I have it working now!

And yes, 56 makes fast work of things that distribute well. Thanks for making the tests run in parallel :)
